src/pages/guides/reference/configuringproxy.md

```diff
@@ -1,7 +1,96 @@
 ## Configuring a Secure Proxy 
 
-Runtime does not expose egress IPs due to security reasons.If customers need a way to securely communicate with their backend services, they can use a proxy in between their backend service and Runtime. 
+Runtime does not expose egress IPs due to security reasons. If customers need a way to securely communicate with their backend services, they can use a proxy in between their backend service and I/O Runtime.
 
-This can be done using the following solution: 
+This can be done by the addition of a proxy component (in this example, an AWS EC2 instance running nginx). The proxy component will have a fixed IP address, therefore enabling the use of an IP allowlist to secure your backend service. The communication between I/O Runtime and the proxy component will be secured via mutual TLS (mTLS) communication. 
```

Comment:
Can we add small diagram that represents the action (Runtime), proxy and backend, that makes visible the mTLS, IP configured on proxy and EC2 instance.